src/main/java/ru/rt/restream/reindexer/binding/cproto/PhysicalConnection.java

```diff
@@ -110,8 +110,8 @@ public PhysicalConnection(String host, int port, String user, String password, S
                 requests.add(new RpcRequest());
                 sequences.add(i);
             }
-            readTaskFuture = scheduler.scheduleWithFixedDelay(new ReadTask(), 0, 1, TimeUnit.MILLISECONDS);
-            writeTaskFuture = scheduler.scheduleWithFixedDelay(new WriteTask(), 0, 1, TimeUnit.MILLISECONDS);
+            readTaskFuture = scheduler.scheduleWithFixedDelay(new ReadTask(), 0, 100, TimeUnit.MICROSECONDS);
```

Comment:
Why do we need to change the delay?

Reply:
In a normal LAN ping is about 0.2ms, and if you send a number of simple requests that don't need responses (mass inserts), you send a request in 0.1 ms and 0.9 ms you await. 
And on tests it really works better.

Reply:
Got it, thanks.